### PR TITLE
Fixes #146. Also adds more events love to dataStore, dataManager, more n...

### LINKED ISSUE #146: Add a disconnectUA method
Now we are using the same unregisterUA method for both:

1.- on disconnect node
2.- when the client wants to REMOVE UA and all associated WA from the server

We should accept both methods

### PR DESCRIPTION
**Pull request by willyaranda**
...eed to come

### CODE REVIEW

File: src/common/datastore.js [hunk 1]
```diff
@@ -95,16 +97,16 @@ var DataStore = function() {
           return;
       });
     });
-  };
+  },
 
   /**
-   * Unregister a node
+   * Node disconnected
```
Comment (frsela) [inline src/common/datastore.js:103]: Disconnects a node (it isn't an event :)

**Comment by frsela** [comment]
Need to be rebased (or check travis fail)
I'll test it before land it. Give me some days to check it.

**Comment by willyaranda** [comment]
Travis failed, but no error was shown :S

**Comment by willyaranda** [comment]
@frsela merge? (I'll fix the comment in the tests PR)

**Comment by frsela** [comment]
We can merge, but travis failed again ... (?) Check it, please

**Pull request merged**